common/djangoapps/student/tests/test_models.py

```diff
@@ -713,6 +714,114 @@ def test_not_enrolled_student_is_enrolled(self):
         assert actual_student.is_active is True
         assert actual_cea.user == student
 
+    @mock.patch.dict('django.conf.settings.FEATURES', {'DISABLE_AUTOENROLL_AFTER_COURSE_END': True})
```

Comment:
These can be simplified:
```suggestion
    @override_settings(FEATURES=dict(settings.FEATURES, DISABLE_AUTOENROLL_AFTER_COURSE_END=True))
```